pocs/camera/camera.py

```diff
@@ -221,12 +231,13 @@ def autofocus(self,
                                       blocking=blocking,
                                       *args, **kwargs)
 
-    def get_thumbnail(self, seconds, file_path, thumbnail_size, keep_files=False):
+    def get_thumbnail(self, seconds, file_path, thumbnail_size, keep_files=False, *arg, **kwargs):
         """
         Takes an image, grabs the data, deletes the FITS file and
         returns a thumbnail from the centre of the iamge.
         """
-        self.take_exposure(seconds, filename=file_path, blocking=True)
+        exposure = self.take_exposure(seconds, filename=file_path, *args, **kwargs)
+        exposure.wait()
```

Comment:
Should we give this a timeout and do something after?

Reply:
This should have a timeout, put possibly the place for it is in the take_exposure() method of the camera class?